docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -1007,6 +1007,19 @@ They can be used for prewhere optimization only if we enable `set allow_statisti
 
     [Count-min](https://en.wikipedia.org/wiki/Count%E2%80%93min_sketch) sketches which provide an approximate count of the frequency of each value in a column.
 
+#### Supported Data Types {#supported-data-types}
+
+| Data Type / Statistics  | count_min | minmax | tdigest | uniq |
```

Comment:
Let's do this:
- We add the supported types to l. 994-1008, e.g.

```md
- `MinMax`
The minimum and maximum column value which allows to estimate the selectivity of range filters on numeric columns.

Supported data types: (U)Int*, Float*, Boolean, Date*
```

Instead of the table in l. 1012, we will have a table for the supported operations: Table rows are all statistics types, table columns are "equal" and "range" (which are the only two classes of predicates that can be evaluated at the moment)

We also need an explanation below the table that "range" means "<, >, and BETWEEN".